docs/tutorial.rst

```diff
@@ -102,11 +104,415 @@ mlf-core is now configured and ready to be used!
 
 For more details including security precautions please visit :ref:`config` and :ref:`github_support`.
 
-
-.. _Github: https://github.com
-
 Creating a mlf-core project
 ------------------------------
 
+mlf-core offers templates for several machine learning libraries. To get an overview of all available machine learning templates run:
+
+.. code-block:: console
+
+    $ mlf-core list
+
+If you want a more detailed overview you can also run:
+
+.. code-block:: console
+
+    $ mlf-core info <template-handle/type/library>
+
+A more detailed overview on all available templates is provided `here <https://mlf-core.readthedocs.io/en/latest/available_templates/available_templates.html>`_.
+In the follow sections we will create and focus on a Pytorch based template identified under the template handle ``mlflow-pytorch``.
+The outlined processes work the same for all other templates.
+
+To create a mlf-core project run:
+
+.. code-block:: console
+
+    $ mlf-core create
+
+You will now be guided interactively through the project creation process.
+mlf-core currently provides two template domains: mlflow and package. Whereas the package templates are designed to create Python packages
+facilitating predictions to be included into complex pipelines, the mlflow templates are used to train deterministic models.
+Hence, select ``mlflow`` and ``pytorch`` afterwards. Enter a project name, a project description, hit enter for the version prompt and selected a license of your choosing.
+MIT and the Apache 2.0 license are common choices. Next, hit the ``y`` button when asked whether you want to create a Github repository and push your code to it.
+If you select ``n`` as in no and create a Github repository manually, mlf-core will not be able to set up required secrets for features such as Docker container building and mlf-core sync.
+Depending on whether you want to create an organization and/or a private repository answer the following prompts with ``y`` or ``n``.
+The project creation process will now end with mlf-core lint verifying the successful creation if your project and the link to your Github repository being printed.
+You are now ready to start training deterministic machine learning models, but first let us have a look at the template's architecture and functionality.
```

Comment:
Concerning line 131-140: This whole text "blob" is hard to read without any line breaks. I'd suggest to insert one or two.